cura/Settings/CuraContainerStack.py

```diff
@@ -387,6 +389,18 @@ def getExtruderPositionValueWithDefault(self, key):
             value = int(Application.getInstance().getMachineManager().defaultExtruderPosition)
         return value
 
+    def getProperty(self, key: str, property_name: str, context = None) -> Any:
+        if property_name == "settable_per_extruder":
```

Comment:
if you're really nit picking on speed and are willing to give in on code readability, you could shorten the "settable_per_extruder" to a shorter string. When trying out various code snippets with timeit I found a speed difference of 0.1 second for 10.000.000 iterations if I shortend it to "spe" ;-)